name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/DABPDFExtractor.java

```diff
@@ -562,6 +569,105 @@ private void addBuySellTaxReturnBlock(DocumentType type)
                 }));
     }
 
+    @SuppressWarnings("nls")
+    private void addDepositRemovalAccountTransaction()
+    {
+        final DocumentType type = new DocumentType("Konto.bersicht", (context, lines) -> {
+            Pattern pCurrency = Pattern.compile("^Verrechnungskonto .* nach Buchungsdatum (?<currency>[\\w]{3})$");
+
+            for (String line : lines)
+            {
+                Matcher m = pCurrency.matcher(line);
+                if (m.matches())
+                {
+                    context.put("currency", m.group(1));
+                }
+            }
+        });
+        this.addDocumentTyp(type);
+
+        Block block = new Block("^(SEPA-Gutschrift|SEPA-Lastschrift) [^Lastschrift].*$");
+        type.addBlock(block);
+        block.set(new Transaction<AccountTransaction>()
+
+                        .subject(() -> {
+                            AccountTransaction transaction = new AccountTransaction();
+                            transaction.setType(AccountTransaction.Type.DEPOSIT);
+                            return transaction;
+                        })
+
+                        // Is type --> "SEPA-Lastschrift" change from DEPOSIT to REMOVAL
+                        .section("type").optional()
+                        .match("^(?<type>SEPA-Gutschrift|SEPA-Lastschrift) [^Lastschrift].*$")
+                        .assign((t, v) -> {
+                            if (v.get("type").equals("SEPA-Lastschrift"))
+                            {
+                                t.setType(AccountTransaction.Type.REMOVAL);
+                            }
+                        })
```

Comment:
`SEPA-Lastschrift` & `SEPA-Gutschrift` sind beides Einzahlungen.
Daher sollte der Typ für `SEPA-Lastschrift` hier nicht verändert werden.

Reply:
Hallo @felschr,
ich verstehe deinen Kommentar nicht.
Wenn erkannt wird, dass die Buchung eine Lastschrift ist, dann wird von `Einbuchung`zu `Ausbuchung`geändert.
Ein Lastschrift ist eine Ausbuchung.... 
```
SEPA-Lastschrift Lastschrift Managementgebühr 29.06.20 53,02
SERVICEENTGELT WELTINVEST RAISIN
SEPA-Lastschrift Lastschrift Managementgebühr 29.06.20 23,05
SERVICEENTGELT WELTINVEST DAB
SEPA-Lastschrift Max Mustermann 13.01.21 400,00
SEPA-Lastschrift
```

Die Einbuchung wäre z.B. dann
```
SEPA-Gutschrift Max Mustermann 05.07.19 15.000,00
SEPA-Überweisung
Einzahlung IOO111462383450 35280593
```

Vielleicht kannst du mir damit auf die Sprünge helfen, wie unterschieden wird zu `Lastschrift `und `Gutschrift`, ggf. an einem Beispiel, damit ich eventuell etwas korrigieren kann.

Reply:
Bei den meisten der `SEPA-Lastschrift`en in meinen Dokumenten handelt es sich um Abbuchungen von dem Referenzkonto für Sparpläne. Somit sind das auch Einzahlungen auf das Depot.
Die `SEPA-Lastschrift Lastschrift Managementgebühr` Einträge sind richtigerweise Gebühren-Abbuchungen und sind ja in Zeile 644 auch schon separat behandelt.
Die `SEPA-Gutschrift`en sind manuelle Überweisungen für Einmalzahlungen, daher die verschiedenen Buchungsweisen.

Hier der relevante Eintrag der `Kontoumsaetze01.txt`:
```
SEPA-Lastschrift Max Mustermann 15.07.19 300,00
SEPA-Lastschrift
WeltInvest Sparplan 000000000
SEPA EndToEndId: 000000000
Gläubiger-ID: DE27ZZZ00000024315
Mandatsref.: SPWELTINVEST0000000000
```

Reply:
Hallo @felschr
lass uns mal TeamViewer nutzen, damit ich das verstehe.
webmaster [at] nirus-online [dot] de
Schick mal bitte eine Einladung... (Headset etc.)